ui/src/app/shared/components/object-editor/object-editor.tsx

```diff
@@ -1,12 +1,13 @@
-import {languages} from 'monaco-editor/esm/vs/editor/editor.api';
 import * as React from 'react';
-import {createRef, useEffect, useState} from 'react';
+import {useEffect, useRef, useState} from 'react';
 import MonacoEditor from 'react-monaco-editor';
```

Comment:
```suggestion
import {useEffect, useRef, useState} from 'react';
import type MonacoEditor from 'react-monaco-editor';
```

same as above comment, `import type` unfortunately fails due to `tslint`

Reply:
Fixed in #12514